app/assets/stylesheets/apply.scss

```diff
@@ -67,6 +67,39 @@
   z-index: -1;
 }
 
+.authorize-icon {
+  font-size: 3em;
+  position: absolute;
+}
+
+.fb-authorize-icon {
+  bottom: 156px;
+  left: 39px;
+}
+
+.google-authorize-icon {
+  bottom: 68px;
+  left: 39px;
+}
+
+.fb-authorize-button {
+  background: #3B5998;
+  width: 100%;
+}
+
+.fb-authorize-button:hover {
+  background: #2F4779;
+}
+
+.google-authorize-button {
+  background: #DD4B39;
+  width: 100%;
+}
+
+.google-authorize-button:hover {
+  background: #B03C2D;
```

Comment:
Color literals like `#B03C2D` should only be used in variable declarations; they should be referred to via variable everywhere else.